front/pages/api/w/[wId]/spaces/[spaceId]/trackers/index.ts

```diff
@@ -17,6 +17,11 @@ import type { SpaceResource } from "@app/lib/resources/space_resource";
 import { TrackerConfigurationResource } from "@app/lib/resources/tracker_resource";
 import { apiError } from "@app/logger/withlogging";
 
+const TRACKER_LIMIT_BY_WORKSPACE = {
+  [md5(PRODUCTION_DUST_WORKSPACE_ID)]: -1,
```

Comment:
The -1 is supposed to be unlimited for us? Don't we need to update the condition below or am I missing something??

Reply:
Ah yes

Reply:
fixed